File: Easy/2833. Furthest Point From Origin/furthest-point-from-origin.py
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:

        cnt_ = moves.count("_")
        cnt_l = moves.count("L")
        cnt_r = moves.count("R")
        
        return cnt_ + max(cnt_l, cnt_r) - min(cnt_l, cnt_r)

File: Easy/2833. Furthest Point From Origin/furthest_point_from_origin.rs
use std::cmp;

impl Solution {
    pub fn furthest_distance_from_origin(moves: String) -> i32 {
        
        let cnt_ = moves.matches("_").count();
        let cnt_l = moves.matches("L").count();
        let cnt_r = moves.matches("R").count();

        let res = cnt_ + cmp::max(cnt_l, cnt_r) - cmp::min(cnt_l, cnt_r);
        res as i32

    }
}
